src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.layout.tsx

```diff
@@ -38,8 +38,7 @@ export function StacksFungibleTokenAssetItemLayout({
   const formattedBalance = formatBalance(amount);
 
   return (
-    //  #4476 TODO - test
-    <Flex as={isPressable ? 'button' : 'div'} outline={0} onClick={onClick} {...(bind as any)}>
+    <Flex onClick={isPressable ? onClick : undefined} {...bind}>
```

Comment:
Problem here is that clickable elements are no longer focusable. How might you create a conditional element?

Reply:
I'll look at refactoring this if that's needed. We could have a wrapper component that is a button if `isPressable` and  `div` otherwise.

Something like this
````
function AssetItemWrapper({isPressable, children}) {

return isPressable ? <styled.button>{children}</styled.button> : <div> {children}</div>

}


```